next.config.js

```diff
@@ -8,4 +8,15 @@ module.exports = withPWA({
   pwa: {
     dest: 'public',
   },
+  async headers() {
+    return [
+      {
+        // match all API routes
+        source: '/api/:path*',
+        headers: [
+          { key: 'Access-Control-Allow-Origin', value: '*' },
```

Comment:
you could have 
```suggestion
          { key: 'Access-Control-Allow-Origin', value: 'https://*' },
```
or if you only wanted the openreferral uk tool to work you could make that explicit with
```suggestion
          { key: 'Access-Control-Allow-Origin', value: 'https://tools.openreferraluk.org' },
```

Reply:
yeah it's just that they say "such as" the API Query Tool. I think I'll drop them a message to see if it really is just that service they'd like to expose it to though.